public/gameInfo.php

```diff
@@ -743,7 +743,7 @@ function submitSetRequest(user, gameID) {
                         echo "<input type='hidden' name='i' value='$gameID'>";
 
                         echo "To remove:";
-                        echo "<select name='m'>";
+                        echo "<select name='m[]' style='resize:auto' multiple>";
                         echo "<option value='0' selected>-</option>";
```

Comment:
probably don't want this anymore (the 0 option)

Reply:
Agree, I removed it.